database-plugins/src/main/java/co/cask/hydrator/plugin/ETLDBOutputFormat.java

```diff
@@ -61,15 +62,44 @@
       Connection connection = getConnection(conf);
       PreparedStatement statement = connection.prepareStatement(constructQuery(tableName, fieldNames));
       return new DBRecordWriter(connection, statement) {
+
+        private boolean emptyData = true;
+
         @Override
         public void close(TaskAttemptContext context) throws IOException {
-          super.close(context);
```

Comment:
Yes, not required, since the implementation below actually contains the exact copy of the parent class' implementation.